### ISSUE TITLE
bigip_command module

### ISSUE THREAD

**Issue by pg0581** [F5Networks/f5-ansible#927]
##### ISSUE TYPE
 - Bug Report

##### COMPONENT NAME
bigip_command

##### ANSIBLE VERSION

```
ansible 2.6.1

```

##### PYTHON VERSION

```
Python 2.7.12
```

##### BIGIP VERSION

-->
```
12.1.2
```

##### LIBRARY VERSIONS

```
f5-sdk==3.0.17
Suds seems to be missing (isn't present in the working version of ansible either 
```

##### SUMMARY
There seems to be an issue with this module and running a check against an LTM in an older version of Ansible works, but in a newer release, it doesn't anymore. (Works in Ansible version 2.5.3 but not in version 2.6.1)

##### STEPS TO REPRODUCE

I've truncated some of this play to keep it shorter
```
    - name: "vs_name"
      prompt: "Enter the virtual server name"
      private: no 

     - name: Pre-check
        bigip_command:
          server: "{{ inventory_hostname }}"
          user: "{{ remote_username }}"
          password: "{{ remote_passwd }}"
          commands:
            - "tmsh list sys global-settings hostname"
            - "tmsh show sys failover"
            - "tmsh show cm sync-status"
            - "tmsh list ltm virtual {{ vs_name }}"
            - "tmsh list ltm pool {{ vs_pool }}"
            - "{{ 'tmsh list ltm rule ' + vs_rule + '' if vs_rule else '' }}"
          validate_certs: no
        delegate_to: localhost
        when: "'active' in Active_LTM['stdout'][0]"
        register: Active_LTM_Pre_Checks

```

##### EXPECTED RESULTS
For the play not to fail if an LTM element hasn't already been configured 

##### ACTUAL RESULTS
```
! Example of the error in version 2.6.1: 
! I can get this play to work in version 2.6.1 if I use the "-M" flag in the "ansible-playbook" command and point it to the directory where I have the "bigip_command.py" file stored" 

The full traceback is:
  File "/tmp/ansible_ARFVcN/ansible_module_bigip_command.py", line 680, in main
    results = mm.exec_module()
  File "/tmp/ansible_ARFVcN/ansible_module_bigip_command.py", line 617, in exec_module
    result = manager.exec_module()
  File "/tmp/ansible_ARFVcN/ansible_module_bigip_command.py", line 409, in exec_module
    changed = self.execute()
  File "/tmp/ansible_ARFVcN/ansible_module_bigip_command.py", line 483, in execute
    self._check_known_errors(responses)
  File "/tmp/ansible_ARFVcN/ansible_module_bigip_command.py", line 523, in _check_known_errors
    raise F5ModuleError(str(resp))

fatal: [x.x.x.x -> localhost]: FAILED! => {
    "changed": false, 
    "invocation": {
        "module_args": {
            "chdir": null, 
            "commands": [
                "tmsh list sys global-settings hostname", 
                "tmsh show sys failover", 
                "tmsh show cm sync-status", 
                "tmsh list ltm virtual test21433", 
                "tmsh list ltm pool test11", 
                ""
            ], 
            "interval": 1, 
            "match": "all", 
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER", 
            "provider": {
                "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER", 
                "server": "x.x.x.x", 
                "server_port": null, 
                "ssh_keyfile": null, 
                "timeout": null, 
                "transport": "rest", 
                "user": "xxxx", 
                "validate_certs": false
            }, 
            "retries": 10, 
            "server": "x.x.x.x", 
  ½         "server_port": null, 
            "transport": "rest", 
            "user": "xxxx", 
            "validate_certs": false, 
            "wait_for": null, 
            "warn": true
        }
    }, 
    "msg": "01020036:3: The requested Virtual Server (/Common/test21433) was not found."
}
```

**Comment by caphrim007** [comment]
@pg0581 i believe this is fixed in ansible 2.7.

**Comment by pg0581** [comment]
@caphrim007 Perfect! Thanks for looking into this. Once we get to using v2.7 I'll let you know if the issue persists.
